Dell/recovery_common.py

```diff
@@ -635,6 +637,21 @@ def parse(fname):
 
     return to_install
 
+def create_grub_entries(rec_type='hdd'):
+    '''Create GRUB entry for dell-recovery during ubiquity installation'''
+    env = os.environ
+    rpart = find_factory_partition_stats()
+    #create the grub entry only when both recovery partition and /cdrom/.disk/info exist
+    if rpart and os.path.exists('/cdrom/.disk/info'):
```

Comment:
So I think the check for `/cdrom/.disk/info` is no longer needed when this is running in the context it's in.  I can't envision a scenario it's still needed.

Reply:
If there is no scenario needed for such check, I surly will remove it.  This check is learned from [here](https://github.com/dell/dell-recovery/blob/master/ubiquity/dell-recovery.py#L190), and the code will check the existence of "/cdrom/.disk/info.recovery" ahead of 99_dell_recovery creation.  Could you please let me know the rationale to do such check in oem-config?

Reply:
It was inserted specifically to only offer to create recovery media if we knew there was a partition that can be recovered from available and usable. Since this is now an install time decision that part is less relevant.